lib/membrane_rtc_engine/endpoints/hls_endpoint.ex

```diff
@@ -300,9 +359,48 @@ if Enum.all?(
           link_builder
           |> to({:keyframe_requester, track.id})
           |> to({:video_parser, track.id})
+          |> then(transcoding_interceptor)
```

Comment:
When transcoding is enabled, `keyframe requester` is useless, but it was left here in order to not put all of the code in lines 360-361 in `create_transcoding_interceptor` or create another interceptor with `keyframe requester` only.

Reply:
I will fix that when rebasing my PR